dataeng/resources/load-events-to-vertica.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if [ "x$NUM_REDUCE_TASKS" = "x" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY*2 ))
+fi
+env
+
+# COMPLETE_OUTPUT_URL=$OUTPUT_URL/$(date +%Y-%m-%d -d "$FROM_DATE")-$(date +%Y-%m-%d -d "$TO_DATE")
+
+${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
+  LoadEventsIntoWarehouseWorkflow --local-scheduler \
+  --interval $(date +%Y-%m-%d -d "$FROM_DATE")-$(date +%Y-%m-%d -d "$TO_DATE") \
+  --n-reduce-tasks $NUM_REDUCE_TASKS \
+  --events-list-file-path $EVENTS_LIST \
+  --warehouse-path $OUTPUT_URL \
+  --credentials $CREDENTIALS \
+  --schema $SCHEMA
+
+# Don't do this for now...
+#   --overwrite
+#  --source s3://edx-analytics-segment/segment-logs/hqawk62tyf/ \
+#  --pattern '.*?/(?P<timestamp>\d{10})\d{3}\..*\.gz' \
+#  --credentials $CREDENTIALS \
+#  --output-root $COMPLETE_OUTPUT_URL \
```

Comment:
These comments are no longer relevant, and can be removed outright.   Same with the COMPLETE_OUTPUT_URL on line 8.